### ISSUE TITLE
delete endpoint is not working using postman

### ISSUE THREAD

**Issue by rahman973** [woocommerce/woocommerce#24932]
Hello,
        I'm facing and getting some error in delete endpoint using any of the base either products or orders. the delete endpoint is not working using the POSTMAN and is throwing error either after using force=true.
all other endpoints like GET, POST both are working fine.
`/wp-json/wc/v3/products/9?force=true`

**Comment by kloon** [comment]
Hi @rahman973 I just tested this with the latest version of WooCommerce and products delete fine without using the force param.

![](https://cld.wthms.co/QjmLuX+)

I suggest you check you are using the correct API credentials and that they are setup with read/write permissions, and also check the user tied to the API credentials have permissions to delete products.

Closing as I am not able to reproduce.
